src/CodeStyle/Core/Analyzers/Options/OptionSetExtensions.cs

```diff
@@ -4,29 +4,29 @@
 
 namespace Microsoft.CodeAnalysis
 {
-    internal static class AnalyzerConfigOptionsExtensions
+    internal static class OptionSetExtensions
     {
-        public static T GetOption<T>(this OptionSet analyzerConfigOptions, Option<T> option)
+        public static T GetOption<T>(this OptionSet optionSet, Option<T> option)
```

Comment:
Should these be renamed? Something like GetEditorConfigOptionByOnlyReadingEditorConfig or something?

Reply:
How is `GetOptionFromEditorConfig` ?

Reply:
These cannot be renamed as they are the codestyle layers implementation of extension methods that exist in the workspace layer. They are here to allow code that is linked into both codestyle and workspace layers to build properly.